src/internal/packager/deploy.go

```diff
@@ -224,7 +224,7 @@ func (p *Packager) deployComponent(component types.ZarfComponent, noImgChecksum
 	}
 
 	if hasRepos {
-		p.pushReposToRepository(componentPath.Repos, component.Repos)
+		_ = p.pushReposToRepository(componentPath.Repos, component.Repos)
```

Comment:
This function returns errors already so this should probably be handled rather than ignored.